docs/core-tech/transaction.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 6
+title: Transaction
+sidebar_label: Transaction
+---
+
+A transaction refers to the act of transferring an entity from one address to another within the network. It is a fundamental operation that enables the exchange of Alephium's native cryptocurrency ALPH, or any other tokens or contract data.
+
+## Prerequisites
+
+It's important that you understand the [UTXO model](/core-tech/stateful-utxo)
+
+## Overview of a transaction
+
+A transaction typically includes the following components:
+
+* Inputs: list of previous unspent outputs, transaction's signatures will prove that those inputs can be spent.
```

Comment:
Specific to P2PKH

Reply:
yeah that's why I introduce with `A transaction typically`, I was not thinking of going into advanced stuff. It should be covered in other docs I think.